xscen/utils.py

```diff
@@ -350,6 +353,8 @@ def clean_up(
     ----------
     ds: xr.Dataset
         Input dataset to clean up
+    xrkwargs: dict
```

Comment:
J'aime pas trop ce nom..
how about `open_dataset_kwargs` ? If there was a way to put this info inside `common_attrs_only`, I would like it even more, because it is not used anywhere else. At least, put them next to each other in the list.